database/mysql.go

```diff
@@ -199,7 +199,7 @@ func mysqlConnection(logger log.Logger, mysqlConfig *MySQLConfig, databaseName s
 // MySQLUniqueViolation returns true when the provided error matches the MySQL code
 // for duplicate entries (violating a unique table constraint).
 func MySQLUniqueViolation(err error) bool {
-	match := strings.Contains(err.Error(), fmt.Sprintf("Error %d: Duplicate entry", mySQLErrDuplicateKey))
+	match := strings.Contains(err.Error(), fmt.Sprintf("Error %d", mySQLErrDuplicateKey))
 	if e, ok := err.(*gomysql.MySQLError); ok {
 		return match || e.Number == mySQLErrDuplicateKey
```

Comment:
looks like [this](https://github.com/moov-io/base/blob/59aa06a665cd185868b743661723ef03d3857191/database/mysql.go#L47) is a dead link now, I think the new one would be https://dev.mysql.com/doc/mysql-errors/8.0/en/server-error-reference.html#error_er_dup_entry if we want to go ahead and update it while we're in here

Reply:
Done